src/mqtt/MQTT.cpp

```diff
@@ -376,12 +376,12 @@ void MQTT::sendSubscriptions()
         const auto &ch = channels.getByIndex(i);
         if (ch.settings.downlink_enabled) {
             hasDownlink = true;
-            std::string topic = cryptTopic + channels.getGlobalId(i) + "/#";
+            std::string topic = cryptTopic + channels.getGlobalId(i) + "/+";
             LOG_INFO("Subscribing to %s\n", topic.c_str());
             pubSub.subscribe(topic.c_str(), 1); // FIXME, is QOS 1 right?
 #ifndef ARCH_NRF52                              // JSON is not supported on nRF52, see issue #2804
             if (moduleConfig.mqtt.json_enabled == true) {
```

Comment:
Re: my comment about subscribing only to the right channel for MQTT JSON uplink, I think this is the conditional it'd need to be added to. In my dynamic language homelands I'd add `&& channels.getGlobalId(i) == "mqtt"` but I think that doesn't work here. Probably easy for one of you, though, haha.